client/pyproject.toml

```diff
@@ -19,15 +19,14 @@ classifiers = [
     "Typing :: Typed",
 ]
 dependencies = [
-  "pyaes~=1.6",
```

Comment:
Being able to run Telethon with Python-only dependencies was a very nice feature. I do not want to see it go.

Can we at least make `pyaes` optional for people who can't or don't want to run `cryptg`? Using `cryptg` as a default is probably a good idea, but removing `pyaes` support is unfortunate.

Reply:
Sure, I'll fix it when I have time. So should cryptg be optional or default?

Reply:
Optional but default should be fine. Telethon has come a long way and CI can build the dependency for a lot of targets, so *most* of the time I'd expect it to work. But for weird or source-only environments, falling back to pyaes is better than nothing.